docs/login-testcase.md

```diff
@@ -0,0 +1,21 @@
+# Login testcase
```

Comment:
I would add information, that this is positive scenario (passing valid login and password). You can create at least one more scenario (with not-valid data + checking the validation of the fields).